rest/route/github.go

```diff
@@ -192,18 +192,6 @@ func (gh *githubHookApi) Run(ctx context.Context) gimlet.Responder {
 				return gimlet.MakeJSONInternalErrorResponder(errors.Wrap(err, "aborting patches"))
 			}
 
-			// if the item is on a commit queue, remove it
-			if err = gh.tryDequeueCommitQueueItemForPR(event.PullRequest); err != nil {
```

Comment:
can you please explain a bit more about why we don't want to dequeue here or check if it's still on the queue?

Reply:
This shouldn't be necessary because `AbortPatchesForGithubPatchData` [already checks for any commit queue items and dequeues them](https://github.com/evergreen-ci/evergreen/blob/1bba8be228ded93d63b0cea395101bd1012fedee/model/patch_lifecycle.go#L933).